server/lib/poller.js

```diff
@@ -80,21 +79,32 @@ class Poller
             for (let access of accesses) {
                 let accountManager = new AccountManager;
                 try {
-                    if (checkAccounts) {
-                        await accountManager.retrieveAccountsByAccess(
-                            access,
-                            false
-                        );
+                    // Only import if lat import did not raise a
+                    // login/parameter error
+                    if (access.canAccessBeUpdated()) {
```

Comment:
How about putting this logic in the account manager, so that we can prevent manual updates when we know the password is wrong?

Also, what happens if the password is wrong, but the user updated it? Should we reset the status?